cpp/src/io/csv/writer_impl.cu

```diff
@@ -411,6 +412,8 @@ void write_csv(data_sink* out_sink,
   // write header: column names separated by delimiter:
   // (even for tables with no rows)
   //
+  cudf::scoped_range range("io::csv::write_csv");
```

Comment:
@mhaseeb123 Is this where you would put the scoped range?

Reply:
Typically, scoped range is used to make a range within a code block. That means, we can have multiple scoped ranges in a single function. On the other hand, `CUDF_FUNC_RANGE` is used to generate a (scoped)range for the entire function, at the top function level.

Reply:
I see that this scoped range is at the top function level so it should be equivalent to `CUDF_FUNC_RANGE`.